.travis.yml

```diff
@@ -20,24 +20,24 @@ script: 'SPEC_OPTS="--format documentation" bundle exec rake validate lint spec'
 matrix:
   fast_finish: true
   include:
-  - rvm: 1.8.7
-    env: PUPPET_GEM_VERSION="~> 3"
-  - rvm: 1.8.7
-    env: PUPPET_GEM_VERSION="~> 3" FUTURE_PARSER="yes"
-  - rvm: 1.9.3
-    env: PUPPET_GEM_VERSION="~> 3"
-  - rvm: 1.9.3
-    env: PUPPET_GEM_VERSION="~> 3" FUTURE_PARSER="yes"
-  - rvm: 2.0.0
-    env: PUPPET_GEM_VERSION="~> 3"
-  - rvm: 2.0.0
-    env: PUPPET_GEM_VERSION="~> 3" FUTURE_PARSER="yes"
-  - rvm: 2.1.9
-    env: PUPPET_GEM_VERSION="~> 3"
-  - rvm: 2.1.9
-    env: PUPPET_GEM_VERSION="~> 3" FUTURE_PARSER="yes"
-  - rvm: 2.1.9
-    env: PUPPET_GEM_VERSION="~> 4"
+    - rvm: 2.1.9
+      env: PUPPET_GEM_VERSION="~> 4.7.0"
+    - rvm: 2.1.9
+      env: PUPPET_GEM_VERSION="~> 4.8.0"
+    - rvm: 2.1.9
+      env: PUPPET_GEM_VERSION="~> 4.9.0"
```

Comment:
Let's get rid of all the minor versions of 4 and just test `~> 4` since if you're running v4, you should be at the latest release.

Reply:
done